693 updated documentation plantnames (#770)
ElektraInitiative/PermaplanT: doc/database/hierarchy.md

(see [Wikipedia](https://en.wikipedia.org/wiki/Hybrid_name)).
So the name does not necessarily say if a plant is a hybrid.

### Rendering the Unique Name

The unique name of a plant needs to be displayed in the following manner:

- Latin name should be displayed in _italics_.
- The cultivar name should be in regular font, enclosed in single quotes (as stored in the database).

For example: _Brassica oleracea italica_ 'Ramoso calabrese'

### Rendering the Complete Name

The complete name of a plant should be displayed as follows:

> - The common name, which appears first, should start with a capital letter (e.g., "tomato" becomes "Tomato").

Reviewer: ```suggestion
- If there is a common name, the common name appears first and should start with a capital letter (e.g., "tomato" becomes "Tomato").
- If it is a plant name from the seed database, the name continues with hyphen `-` and the additional name.
```

> - This is followed by the unique name in italics, and finally the cultivar name in single quotes.

Reviewer: ```suggestion
- This is followed by the unique name in brackets, unique name as described above in `Rendering the Unique Name`.
```

The general format is:

`Common_name (_unique\_name_ 'cultivar')`

In cases where the common name is not available in the current language, it should be displayed as:

`_unique\_name_ 'cultivar'`

> Currently, the cultivar is a part of the unique name.

Reviewer: ```suggestion
The cultivar is a part of the unique name.
```

(see [Wikipedia](https://en.wikipedia.org/wiki/Hybrid_name)).
So the name does not necessarily say if a plant is a hybrid.

### Rendering the Unique Name

The unique name of a plant needs to be displayed in the following manner:

- Latin name should be displayed in _italics_.

> - The cultivar name should be in regular font, enclosed in single quotes (as stored in the database).

Reviewer 2: This isn't your fault, but since you are already updating this document could you maybe explain further above what the cultivar is?
That might be useful for new project contributors.

Reviewer: There is an explanation of Cultivar in doc/architecture/12glossary.md

For example: _Brassica oleracea italica_ 'Ramoso calabrese'

### Rendering the Complete Name

The complete name of a plant should be displayed as follows:

- If there is a common name, the common name appears first and should start with a capital letter (e.g., "tomato" becomes "Tomato").
- If it is a plant name from the seed database, the name continues with hyphen `-` and the additional name.
- This is followed by the unique name in brackets, unique name as described above in `Rendering the Unique Name`.

> The general format is:

Reviewer 2: Again nothing that you did wrong.
We sometimes only rely on the common name (e.g. in the plant label feature that was just merged).
It would be useful if there was a short paragraph explaining when shortened names are appropriate.

(Also maybe not as part of this PR, but as a follow-up.)

Author: I added that based on Markus proposal.

Common name - additional name (_unique name_ 'cultivar name')

In cases where the common name is not available in the current language, it should be displayed as:

_unique name_ 'cultivar name' - additional name

The cultivar is a part of the unique name.

For example:

- Italian broccoli - violett (_Brassica oleracea italica_ 'Ramoso calabrese')
- Italian broccoli - violett (_Brassica oleracea italica_)
- Italian broccoli (_Brassica oleracea italica_)
- _Brassica oleracea italica_ 'Ramoso calabrese' - violett
- _Brassica oleracea italica_ 'Ramoso calabrese'
- _Brassica oleracea italica_ - violett
- _Brassica oleracea italica_

> 

Reviewer: @badnames does this help?

```suggestion

Usage of complete name:

- The common name alone is enough for overview, e.g. when you see all labels in the map editor at once.
- The complete name should be always be given if the user wants to know precisely which plant, e.g. in search results or in plant details.
- The additional name needs to be additionally given if a plant is connected with a seed.

```

Author: Added that and extended it a bit

Reviewer: Great, looking forward to the other changes I already proposed earlier to get this merged&done.

Author: should all be implemented now

(see [Wikipedia](https://en.wikipedia.org/wiki/Hybrid_name)).
So the name does not necessarily say if a plant is a hybrid.

### Rendering the Unique Name

The unique name of a plant needs to be displayed in the following manner:

- Latin name should be displayed in _italics_.
- The cultivar name should be in regular font, enclosed in single quotes (as stored in the database).

For example: _Brassica oleracea italica_ 'Ramoso calabrese'

### Rendering the Complete Name

The complete name of a plant should be displayed as follows:

- If there is a common name, the common name appears first and should start with a capital letter (e.g., "tomato" becomes "Tomato").
- If it is a plant name from the seed database, the name continues with hyphen `-` and the additional name.

> - This is followed by the unique name in brackets, unique name as described above in `Rendering the Unique Name`.

Reviewer: ```suggestion
- This is followed by the unique name in brackets, unique name as described above in [Rendering the Complete Name](###rendering-the-complete-name).
```

The general format is:

> Common name - additional name (_unique name_ 'cultivar name')

Reviewer: cultivar is part of unique name:

```suggestion
Common name - additional name (_unique name_)
```

In cases where the common name is not available in the current language, it should be displayed as:

> _unique name_ 'cultivar name' - additional name

Reviewer: ```suggestion
_unique name_ - additional name
```

The cultivar is a part of the unique name.

For example:

- Italian broccoli - violett (_Brassica oleracea italica_ 'Ramoso calabrese')
- Italian broccoli - violett (_Brassica oleracea italica_)
- Italian broccoli (_Brassica oleracea italica_)
- _Brassica oleracea italica_ 'Ramoso calabrese' - violett
- _Brassica oleracea italica_ 'Ramoso calabrese'
- _Brassica oleracea italica_ - violett
- _Brassica oleracea italica_

### Usage of plant names

- The common name alone is enough for overview, e.g. when you see all labels in the map editor at once.
  - If no common name exists, it's acceptable to display the unique name as shown in [Rendering the Complete Name](###rendering-the-complete-name).

> - The complete name should be always be given if the user wants to know precisely which plant, e.g. in search results or in plant details.

Reviewer: ```suggestion
- The complete name should always be given if the user wants to know precisely which plant she is dealing with, e.g. in search results or in plant details.
```

> - The additional name created when adding a new seed needs to be provided additionally if a plant is connected with a seed.

Reviewer: ```suggestion
- The additional name created when adding a new seed needs to be provided as explained above if a plant is connected with a seed.
```
